web/modules/custom/joinup_search/config/install/search_api.index.published.yml

```diff
@@ -60,7 +60,7 @@ field_settings:
         - 'entity:rdf_entity/og_audience'
         - 'entity:rdf_entity/field_isr_is_version_of'
   entity_author:
-    label: Author
+    label: 'Aggregated field'
```

Comment:
This is a regression, it should say author. I think this is caused by an unlucky merge.